src/frontend/gatsby/src/components/SearchPage/SearchResults.jsx

```diff
@@ -41,14 +64,15 @@ const SearchResults = ({ text, filter }) => {
     return (
       <Container className={classes.searchResults}>
         <Box className={classes.root} boxShadow={2} marginTop={10}>
-          <ListSubheader>
-            <Typography variant="h1" color="secondary" className={classes.title}>
-              <Grid container className={classes.error}>
-                {' '}
+          <div className={classes.errorBackground}>
+            <h1>
+              500{' '}
```

Comment:
I don't think the `500` is useful (it might be some other error, for example, 503).  Why don't we switch this to some user-friendly text, maybe `Search Error` or something?